stdlib/io.pyi

```diff
@@ -48,13 +48,13 @@ class IOBase:
     def closed(self) -> bool: ...
     def _checkClosed(self, msg: str | None = ...) -> None: ...  # undocumented
 
-class RawIOBase(IOBase):
+class RawIOBase(IOBase, BinaryIO):
     def readall(self) -> bytes: ...
     def readinto(self, __buffer: WriteableBuffer) -> int | None: ...
     def write(self, __b: ReadableBuffer) -> int | None: ...
     def read(self, __size: int = ...) -> bytes | None: ...
 
-class BufferedIOBase(IOBase):
+class BufferedIOBase(IOBase, BufferedIOBase):
```

Comment:
I don't know what you are trying to do here, but `class Foo(Bar, Foo):` doesn't make any sense, and that's what pyright complains about.

Reply:
Ugh, it seems my dyslexia yesterday was making me read a totally different word, sometimes my brain gets the visual mental cache wrong :sweat_smile:

This should of course should be `BinaryIO`.